home/apps.py

```diff
@@ -0,0 +1,12 @@
+from django.apps import AppConfig
+
+
+class HomeAppConfig(AppConfig):
+    name = "home"
+
+    def ready(self):
+        from wagtail.models.reference_index import ReferenceIndex
+
+        from home.models import OrderedContentSet
+
+        ReferenceIndex.register_model(OrderedContentSet)
```

Comment:
Ah interesting, I thought that snippets were automatically registered: https://docs.wagtail.org/en/latest/advanced_topics/reference_index.html#configuration

Reply:
Yeah, looks like I can remove that.